src/main/kotlin/com/example/commerce/payment/service/PaymentService.kt

```
@@ -0,0 +1,186 @@
+        }
+
+        val readyResponse = try {
+            kakaoPayClient.ready(
+                orderId = payment.orderId,
+                orderName = order.name,
+                orderKey = order.key,
+                userId = payment.userId,
+                quantity = 0,
+                totalAmount = payment.paidAmount
+            )
+        } catch (e: HttpClientErrorException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)